src/index.tsx

```diff
@@ -195,21 +195,28 @@ export default function AxeMode({ children, disabled }: AxeModeProps) {
     }
 
     if (childrenRef.current) {
-      if (idleId) {
+      if (idleId && 'cancelIdleCallback' in window) {
```

Comment:
I think this is a good approach with `requestIdleCallback`. I think bundling a polyfill for an experimental API is a bit much for a relatively minor optimization.